internal/session/session.go

```diff
@@ -381,7 +391,7 @@ func newCert(ctx context.Context, wrapper wrapping.Wrapper, userId, jobId string
 	if len(addresses) == 0 {
 		return nil, nil, errors.New(ctx, errors.InvalidParameter, op, "missing addresses")
 	}
-	pubKey, privKey, err := DeriveED25519Key(ctx, wrapper, userId, jobId)
+	pubKey, privKey, err := ed25519.GenerateKey(nil)
```

Comment:
Using a `nil` random reader defaults the reader to `crypto/rand.Reader`

Reply:
Correct, but don't do this. :-D Pass in the SecureRandomReader from the base server (Controller in this case), which will require a bit of plumbing, but it's a forward-looking thing.

Reply:
Will do